client/src/app/unauthorized.component.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, it } from 'vitest';
+
+import { render, screen, userEvent, waitFor } from '~/test/utils';
+
+import { Unauthorized } from './unauthorized.component';
+
+describe('Unauthorized', () => {
+  it('should render unauthorized component', () => {
+    render(<Unauthorized />);
+
+    expect(screen.getByRole('heading', { name: /Unauthorized/i })).toBeInTheDocument();
```

Comment:
by using the `//i` syntax, which means regex case agnostic, then you shouldn't capitalize anything, keep it all lowercase. That is how Kent Dodds and other bloggers do it.